### ISSUE TITLE
androidx.compose-bom v2024.09.00 poll rendering regression

### ISSUE THREAD

**Issue by frebib** [element-hq/element-x-android#3566]
### Steps to reproduce

Vote on a poll and notice it renders weirdly:
https://github.com/element-hq/element-x-android/commit/9756afc5d1d2859ca1dbc94827736130ea0120f6#diff-137a3fe986cdfd070ed5df92df53e3b8fe61d01fbf579838fae3add428d49a38

#### What did you expect?
No changes

#### What happened instead?
Wonky rendering

### Will you send logs?

No

### Are you willing to provide a PR?

No

**Comment by frebib** [comment]
Seems related https://issuetracker.google.com/issues/365033302

**Comment by bmarty** [comment]
Yes, detected during the PR review https://github.com/element-hq/element-x-android/pull/3399#discussion_r1764641667

**Comment by jmartinesp** [comment]
As a possible workaround they say we could use this:

```
LinearProgressIndicator(
  progress = { animatedProgress },
  trackColor = MaterialTheme.colorScheme.surfaceVariant,
  strokeCap = StrokeCap.Butt,
  gapSize = 0.dp,
  drawStopIndicator = null
)
```

**Comment by jmartinesp** [comment]
https://github.com/element-hq/element-x-android/pull/3583 should fix this using the workaround provided by Google (https://github.com/element-hq/element-x-android/issues/3566#issuecomment-2388397829).
